dataFields/ABFieldTextFormulaCore.js

```diff
@@ -548,29 +556,29 @@ function getNumberToWords(number) {
 }
 
 function getDateDayOfWeekName(abFactory, date) {
-   // var localizeDT = moment(date);
+   // const localizeDT = moment(date);
    // localizeDT.locale(AD.lang.currentLanguage);
    // return localizeDT.format("dddd");
 
    return abFactory.toDateFormat(date, {
       format: "dddd",
-      localeCode: AD.lang.currentLanguage,
+      localeCode: abFactory.Config._config.user.languageCode,
    });
 }
 
 function getDateMonthOfYearName(abFactory, date) {
-   // var localizeDT = moment(date);
+   // const localizeDT = moment(date);
    // localizeDT.locale(AD.lang.currentLanguage);
    // return localizeDT.format("MMMM");
 
    return abFactory.toDateFormat(date, {
       format: "MMMM",
-      localeCode: AD.lang.currentLanguage,
+      localeCode: abFactory.Config._config.user.languageCode,
```

Comment:
`AB.Account.language()`

Reply:
just to point out ...

Anytime you reference a private inner value of an object, that is a `no-no`.  For example: `Config._config`  the `_` prefix is supposed to warn you that this is private.  You should be using an exposed method to access any of that data.